data/.schema.yaml

```diff
@@ -180,6 +181,14 @@ properties:
                 - type: boolean
                 - type: string
                   constant: unknown
+            lineage:
+              description: Lineage information of the disease for the participant.
+              type: string
+            confirmation_gap:
+              description: Confirmation occurs how many days after symptom onset. A negative value indicate that the confirmation occurs before sympotom onset.
+              oneOf:
+                - type: number
+                - const: unknown
```

Comment:
It feels like it might be something that's unique to this particular dataset. Do you think this is worth having in the schema?

Reply:
Yes, this is unique. Will the validation pass, if the schema does not define it? If so, I can remove the `confirmation_gap`.